homeassistant/components/camera/__init__.py

```diff
@@ -69,6 +73,11 @@
     vol.Required(ATTR_FILENAME): cv.template
 })
 
+CAMERA_SERVICE_PLAY_STREAM = CAMERA_SERVICE_SCHEMA.extend({
+    vol.Required(ATTR_MEDIA_PLAYER): cv.comp_entity_ids,
```

Comment:
We need a validator to ensure this is an media player entity(s)

Reply:
I'm not sure why.  The base service schema doesn't ensure `entity_id` is a camera entity.  If it's not a media player, the `play_media` service won't work and the user will need to adjust.

Reply:
I am not afraid `entity_id`, since all services followed same pattern. However `media_player` is a special usage for this service, and we should fail the service call as earlier as possible if user inputed wrong media player entity id

Reply:
We have `cv.entities_domain(DOMAIN)`, however, you will need to still allow `ENTITY_MATCH_ALL` (which is a valid value for `cv.comp_entity_ids`.

Alternative, you can drop `cv.comp_entity_ids` for now, and just make it `cv.entities_domain(DOMAIN)`